Sources/Web5/Crypto/JOSE/JWT.swift

```diff
@@ -90,4 +91,64 @@ public struct JWT {
             )
         )
     }
+
+    public struct ParsedJWT {
+        let header: JWS.Header
+        let payload: [String: AnyCodable]
+
+        public init(
+            header: JWS.Header,
+            payload: [String: AnyCodable]
+        ) {
+            self.header = header
+            self.payload = payload
+        }
+    }
+
+    public static func parse(jwtString: String) throws -> ParsedJWT {
+        let parts = jwtString.components(separatedBy: ".")
+
+        guard parts.count == 3 else {
+            throw Error.verificationFailed("Malformed JWT. Expected 3 parts. Got \(parts.count)")
+        }
+        
+        let base64urlEncodedJwtHeader = String(parts[0])
+        let base64urlEncodedJwtPayload = String(parts[1])
+        let _ = String(parts[2])
```

Comment:
do we need to explicitly state this if we're not gonna use it? (i'm making an assumption that naming something `_` means it's to be discarded